src/transaction/transaction_factory.cpp

```diff
@@ -0,0 +1,47 @@
+#include <utility>
+
```

Comment:
At this point, I'm not sure whether defining only builders (no factory) would have been friendly-enough for developers.